blocks/editable/style.scss

```diff
@@ -56,6 +56,19 @@
 	}
 }
 
+.editor-visual-editor__block:not( .is-selected ) .blocks-editable.drop-cap-true {
```

Comment:
Also, I was expecting the style to trigger as soon as I toggle the inspector control. Why is it only for not selected blocks?